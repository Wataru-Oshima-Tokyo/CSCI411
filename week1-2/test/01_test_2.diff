1,30c1,2
< Number of nodes and number of edges: 
< Add 7 edges: 
< 
< Original
< 0 1 2 3 4 5 
< 3 4 
< 1 
< 2 5 
< 5 
< 4 
<  Transposed 
< 0 1 2 3 4 5 
< 2 
< 3 
< 1 
< 1 5 
< 3 4 
< 0 0
< 1 0
< 2 0
< 3 0
< 4 1
< 5 1
< sccs size: 2
< id: 0 size: 3
< In: 0
< Out 1
< id: 1 size: 2
< In: 1
< Out 0
---
> Number of nodes and number of edges: 
> Add 7 edges: 
