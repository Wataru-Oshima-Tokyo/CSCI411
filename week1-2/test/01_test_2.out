Number of nodes and number of edges: 
Add 7 edges: 
|A| = 3, |B| = 2, |C| = 0